packages/patternlab/styleguide/source/_patterns/02-molecules/banner-credit.twig

```diff
@@ -1,10 +1,11 @@
-<div class="ma__banner-credit">
-  <div class="ma__banner-credit__container">
-    <div class="ma__banner-credit__icon">
+<div class="ma__banner-credit" aria-hidden="true">
+  <dl class="ma__banner-credit__container">
+    <dt class="ma__banner-credit__icon">
+      <span class="ma__visually-hidden">Credit for the banner image</span>

```

Comment:
Keep this structure for now in case some other use cases besides the banner image for this component comes up.  Then, at least the base is there even it needs some conditions to add.